llvm/test/CodeGen/RISCV/rvv/vadd-vv.ll

```diff
@@ -0,0 +1,14 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
```

Comment:
This file should be moved to `test/CodeGen/RISCV/GlobalISel` somewhere? It looks like you are either trying to test globalisel and stop after IRTranslation, or you are trying to test global isel LLVM all the way down RVV ASM. In the former case, you need to add -global-isel because this is using SDAG without it. In the latter case, you should drop the stop-after=irtranslator.